### PR TITLE
feat: trigger send from existing emojis

### LINKED ISSUE #2060: Add ability to send same emoji at nft item detail
When there is already emoji at nft, you want to let the user click on the same to send it 

image

Test case - https://kodadot.xyz/rmrk/gallery/10843795-0a8ce195286c168f19-DONKEY-DONKEY_GANG_473-0000000000000473

### PR DESCRIPTION
**Pull request by kkukelka**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Do a quick check before the merge.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

https://user-images.githubusercontent.com/17953978/150839421-83baa90e-d518-4baa-b7d0-3dbffb5d5886.mov

### What's new?

- [x] PR closes #2060
- [x] now you can click to existing emojis to send emote

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Optional

- [x] I've tested it at </rmrk/gallery/8774567-dac5c7f54029d0e73c-HXAI8-CAPETAIN_AMERICA-0000000000000020>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### Had issue bounty label ?

- [x] Fill up your KSM address: [Payout](https://kodadot.xyz/transfer/?target=EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI, a screenshot is best to understand changes for others.

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
wouldn't it be better to replace `<b-tag>` with `<b-button>` from EmotionList component?

NB: There is a Bulma Helper (`is-clickable`) for cursor: pointer

**Comment by kkukelka** [comment]
> wouldn't it be better to replace `<b-tag>` with `<b-button>` from EmotionList component?
> 
> NB: There is a Bulma Helper (`is-clickable`) for cursor: pointer

I refactored it so that we use bulma-class,
however using `<b-button>` isn't just an easy replace. i would have to overwrite some of its css via !important to make it look the tag.
accessibility-wise i would agree on using button, but maybe out of scope for this ticket?

File: components/rmrk/Gallery/EmotionList.vue [hunk 1]
```diff
@@ -2,14 +2,14 @@
   <div
     v-show="emotes"
     class="columns is-mobile is-multiline is-gapless nft-emotion__main">
```
Comment (roiLeo) [inline components/rmrk/Gallery/EmotionList.vue:4]:
```suggestion
    class="buttons mb-2">
```

File: components/rmrk/Gallery/EmotionList.vue [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
-    <div
-      v-for="emoji in emotes"
-      :key="emoji.key"
-      class="column nft-emotion__column">
-      <b-tag id="emoji-box" type="is-dark is-large">
+    <div v-for="emoji in emotes" :key="emoji.key" class="column">
+      <b-button
```
Comment (roiLeo) [inline components/rmrk/Gallery/EmotionList.vue:6]:
```suggestion
      <b-button v-for="emoji in emotes" :key="emoji.key"
```

File: components/rmrk/Gallery/EmotionList.vue [hunk 3]
```diff
@@ -2,14 +2,14 @@
...
+        id="emoji-box"
```
Comment (roiLeo) [inline components/rmrk/Gallery/EmotionList.vue:7]:
```suggestion
```
not a fan as id should be unique

File: components/rmrk/Gallery/EmotionList.vue [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
+        type="is-dark"
```
Comment (roiLeo) [inline components/rmrk/Gallery/EmotionList.vue:8]:
```suggestion
        type="is-outlined"
```

File: components/rmrk/Gallery/EmotionList.vue [hunk 5]
```diff
@@ -2,14 +2,14 @@
...
+        @click="$emit('selected', emoji.key)">
         {{ String.fromCodePoint(parseInt(emoji.key, 16)) }}
         <span class="nft-emotion-tag__count">{{ emoji.count }}</span>
```
Comment (roiLeo) [inline components/rmrk/Gallery/EmotionList.vue:11]:
```suggestion
        <span>{{ emoji.count }}</span>
```
I don't think we need the class for now, could be changed with spacing helper

File: components/rmrk/Gallery/EmotionList.vue [hunk 6]
```diff
@@ -2,14 +2,14 @@
...
-      </b-tag>
+      </b-button>
     </div>
```
Comment (roiLeo) [inline components/rmrk/Gallery/EmotionList.vue:13]:
```suggestion
```
unused div wrapper

**Review by roiLeo — COMMENTED** [review]
I've try to change with component only solution let me know what you think.
The css will no longer be necessary.

**Comment by kkukelka** [comment]
@roiLeo liked your approach and its working, even prettier than before because of the spacing between elements
still couldn't get rid of the `emoji-box` class since i need to overwrite some of the button classes

**Review by roiLeo — APPROVED** [review]
lgtm

#### With emoji-box
Screenshot 2022-01-25 at 13-35-35 cAPEtain America

#### Without
Screenshot 2022-01-25 at 13-36-06 cAPEtain America

#### Without & with is-medium btn
Screenshot 2022-01-25 at 13-39-55 cAPEtain America

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 172.43 USD/KSM ~ 0.58 $KSM
🧗 [EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW](https://kusama.subscan.io/account/EqdyzrzVmeHwMdMwvPeCMnNdbuQDbD3YrjY93xq9Ln3jUGW)
🔗 [0xae4d7f4fc053588c057249dd05f413c903e7c662639548be1b2897776aab7b21](https://kusama.subscan.io/extrinsic/0xae4d7f4fc053588c057249dd05f413c903e7c662639548be1b2897776aab7b21)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by vikiival** [comment]
Lovely ❤️

**Pull request merged**